indra/statements.py

```diff
@@ -437,44 +467,21 @@ class ActivityModification(Statement):
     """Statement representing the activation of a protein as a result
     of a residue modification"""
 
-    def __init__(self, monomer, mod, mod_pos, relationship, activity,
+    def __init__(self, monomer, mod, relationship, activity,
                  evidence=None):
         super(ActivityModification, self).__init__(evidence)
         self.monomer = monomer
-        # This means that one of mod and mod_pos are a list, but not both,
-        # which should raise a ValueError
-        if isinstance(mod, list) and len(mod) > 1 and mod_pos is None:
-            raise ValueError('If more than one modification is specified then '
-                             'mod_pos must also be specified and cannot be '
-                             'None.')
-        elif isinstance(mod, list) != isinstance(mod_pos, list):
-            raise ValueError('If mod or mod_pos are provided as lists they '
-                             'must be matched.')
-        elif isinstance(mod, list) and isinstance(mod_pos, list) and \
-                len(mod) != len(mod_pos):
-            raise ValueError('If mod and mod_pos are lists, then they must be '
-                             'the same length.')
-        elif isinstance(mod, list) and isinstance(mod_pos, list) and \
-                len(mod) == len(mod_pos):
-            # Set the fields to be the lists, but make sure to stringify
-            # any entries in the mod_pos list that might be ints
-            self.mod = mod
-            self.mod_pos = [str(mp) for mp in mod_pos]
-        elif isinstance(mod, basestring) and \
-                (isinstance(mod_pos, int) or isinstance(mod_pos, basestring)):
-            self.mod = [mod]
-            self.mod_pos = [str(mod_pos)]
-        elif isinstance(mod, basestring) and mod_pos is None:
+        # Turn mod conditions into a list of a single one is given
+        if isinstance(mod, ModCondition):
```

Comment:
Maybe file an issue for ActivityModifications where the modcondition is None, implying that the agent is constitutively active. This could be useful for indicating that mutants are active while the WT proteins require modifications to be active.